src/main/java/frc/team2412/robot/util/gyroscope/Gyroscope.java

```diff
@@ -5,6 +5,8 @@
 public abstract class Gyroscope {
 	private Rotation2d adjustmentAngle = Rotation2d.fromRotations(0);
 	private boolean inverted = false;
+	private boolean simulated = false;
```

Comment:
If you're looking for a way to check if we're running in sim, there's the `Robot.isSimulation()` method here: https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/RobotBase.html#isSimulation().